resources/views/index.blade.php

```diff
@@ -110,24 +113,6 @@
       <div class="row">
         <div class="col-md-12">
           <h1>SPONSORS</h1>
-          <div class="row">
```

Comment:
can you hide the sponsors section entirely for now? Looks weird for there to be none, especially if sponsors visit the site.

Specifically, add a `display:none` to   `<div class="sponsors_wrap">` on Line 108

Reply:
![image](https://user-images.githubusercontent.com/22565214/42199931-f3bf27f4-7e45-11e8-8d58-6637e9696c40.png)
Removed sponsors from nav-bar

![image](https://user-images.githubusercontent.com/22565214/42199945-feee4fd8-7e45-11e8-8d54-7e81808cc2ab.png)
Removed sponsors section from main page

Reply:
![capture1](https://user-images.githubusercontent.com/22565214/42739471-b58bb298-884c-11e8-803f-b474d682e1a8.PNG)
Added community tab to the nav-bar that appears after a user logs in.

![capture2](https://user-images.githubusercontent.com/22565214/42739477-c95525a2-884c-11e8-961c-96be31012e6a.PNG)
Allow mentors and organizers to view the community page when the community link is clicked by changing the middleware requirement to be Phase 1 instead of Phase 2.